engine/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ if (MSVC)
     if (CMAKE_CXX_FLAGS MATCHES "/W[0-4]") # Force to always compile with W4
         string(REGEX REPLACE "/W[0-4]" "/W4" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
     else()
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4 /wd4201 /wd4244")

```

Comment:
[Warning C4201](https://learn.microsoft.com/ko-kr/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4201?view=msvc-170)

[Warning C4244](https://learn.microsoft.com/ko-kr/cpp/error-messages/compiler-warnings/compiler-warning-levels-3-and-4-c4244?view=msvc-170)